allennlp/models/semantic_role_labeler.py

```diff
@@ -175,6 +183,14 @@ def tag(self, text_field: TextField, verb_indicator: IndexField) -> Dict[str, An
 
         return {"tags": tags, "class_probabilities": predictions.numpy()}
 
+    def get_metrics(self, reset: bool = False):
+        # TODO(Mark): This won't play well with the metrics in Trainer, see how Matt wants it.
```

Comment:
Ah, having now seen the metric code, no it's not.  I'd flatten out the dictionary either in the `Metric` class or here (probably in the `Metric` class), so there's only one `Dict[str, float]`.